.github/workflows/opened-pr-test.yml

```
@@ -12,7 +12,97 @@
+          '
+          
+          - name: Elasticsearch connection
+            run: |
+              curl -v http://localhost:9200
+
+      - name: Elasticsearch connection
+        run: |
+          curl http://127.0.0.1:9200
+
+      - name: MySQL connection health check
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:8:6: Double quote to prevent globbing and word splitting

(shellcheck)